### PR TITLE
Add tf_hostname to tf_admin module

### PR DESCRIPTION
**Pull request by drama17**
## What
* use `tf_hostname` to define the `admin` host in the `prod.autopay.cc` domain

## Why
* Need valid hostname for admin host

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -22,3 +18,6 @@ output "role" {
   value = "${module.instance.role}"
 }
 
+output "domain_name" {
```
Comment (osterman) [inline outputs.tf]: Just call it `hostname` or `fqhn`

**Review by osterman — CHANGES_REQUESTED** [review]
Looks good, just one tweak for output variable name.

File: outputs.tf (hunk 1 above)
Comment (drama17) [reply outputs.tf]: Renamed

**Review by osterman — CHANGES_REQUESTED** [review]
tf_domain does not belong here

**Review by osterman — APPROVED** [review]

File: main.tf [hunk 2]
```diff
@@ -63,3 +63,14 @@ module "instance" {
...
   ansible_arguments   = "${var.ansible_arguments}"
 }
+
+module "dns" {
+  source    = "git::https://github.com/cloudposse/tf_hostname.git?ref=tags/0.1.0"
+  namespace = "${var.namespace}"
+  name      = "${var.name}"
+  stage     = "${var.stage}"
+  zone_id   = "${module.ap_domain.zone_id}"
+  ttl       = 60
```
Comment (const-bon) [inline main.tf:73]: Add `ttl` as variable.

File: outputs.tf [hunk 3]
```diff
@@ -22,3 +18,6 @@ output "role" {
   value = "${module.instance.role}"
 }
 
+output "fqdn" {
```
Comment (osterman) [inline outputs.tf:21]: This is `fqhn` not `fqdn`

**Pull request merged**